packages/mui-material-next/src/FormControl/FormControlContext.ts

```diff
@@ -8,26 +8,26 @@ type ContextFromPropsKey =
   | 'fullWidth'
   | 'hiddenLabel'
   | 'margin'
-  | 'onBlur'
-  | 'onFocus'
   | 'required'
   | 'size'
   | 'variant';
 
-export interface FormControlState extends Pick<FormControlProps, ContextFromPropsKey> {
+export interface FormControlContextValue extends Pick<FormControlProps, ContextFromPropsKey> {
```

Comment:
I agree with the name change. Could this be a breaking change for developers who are using this interface? If so, we should document it in `packages/mui-material-next/migration.md`

Reply:
Added~